### ISSUE TITLE
On invoking 'Copy brush name' button, Screen reader is announcing the 'Copied to clipboard' success message twice.

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2083]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: 24H2 (OS Build 26100.4315)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)
AT: Screen reader

Pre-Requisite:
Turn on Narrator with shortcut key "win+ctrl+Enter" key.

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'Color' in search box and hit enter.
Navigate to 'Color' card and activate it.
Navigate to 'Text' list item and select it.
Navigate to 'Copy brush name' button and invoke it. 
Now listen to the screen reader announcement
Actual Result:
On invoking 'Copy brush name' button, Screen reader is announcing the 'Copied to clipboard' success message twice. It is announcing as "Brush name copied to clipboard, Copied to clipboard"

Note: Issue is observed in entire component

Expected Result:
On invoking 'Copy brush name' button, Screen reader should announce the success message only once. It should announce like 'Brush name copied to clipboard'.

User Impact:
Users with visual impairments who rely on screen reader will face difficulties, if copied success message information announcing twice

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

### Windows version

Windows 11 24H2 (26100)
